client/state/automated-transfer/constants.js

```diff
@@ -9,6 +9,8 @@ export const transferStates = {
 
 export const eligibilityHolds = {
 	MULTIPLE_USERS: 'MULTIPLE_USERS',
+	NO_BUSINESS_PLAN: 'NO_BUSINESS_PLAN',
+	NO_VIP_SITES: 'NO_VIP_SITES',
```

Comment:
Can you add these to the API mapping as well?

`state/data-layer/wpcom/sites/automated-transfer/eligibility/index.js`